### ISSUE TITLE
inactive parks

### ISSUE THREAD

**Issue by Tyedye** [amtgard/ORK3#84]
There should be something on the park page itself that indicates when it is inactive, for casual users. 

Theoretically you should be able to see it when you go to park, admin, configure park, but even with inactive parks that lists them as active. (see CK, Eclipse)

Additionally, inactive parks, while not on the main atlas, are on the kingdom specific atlas. Perhaps a toggle on the main atlas, to chose weither or not to show them would solve this, so that when starting a new park you can view dead parks in the area, but when looking for a park to visit, you will not be given false hope.

**Comment by awozniak** [comment]
Seems like you could deduce active/inactive from attendance logs.  Or show last time anyone attended on the park pages.

**Comment by BenOvermyer** [comment]
I'm in favor of a time-based update of the Active/Inactive flag. That'd be easy enough to add.

**Comment by Tyedye** [comment]
I am not.  I know I am way behind updating, but I would like to update
based on state of amtgard responces. Using the ork is not actually required.

**Comment by BenOvermyer** [comment]
Relying on manual processes opens the system up to human error, though. If using the ORK is not required, then where is the single source of truth for the data?

**Comment by awozniak** [comment]
Simply putting the last attendance date up on the park page would solve the first part of the original poster's complaint.

**Comment by Hunteil** [comment]
Deducing last active member signins won't work. Because if my park would fall apart. I'd go to another park and sign in there. If I don't move my profile over to their park b/c...lets say I'm driving an hour to get there. Then that attendance will show up in my home parks active feedback.

So we'd have to make sure the program is detecting the attendance at their park in question is needed too.

**Comment by BenOvermyer** [comment]
So are these the rules?

* A _member_ is active if: they have a recent (last 6 months) sign-in at an event
* A _park_ is active if: it has had members (of any park) sign-in at an event it ran within the last 6 months
* A _kingdom_ is active if: it has not been deliberately marked inactive

**Comment by Glenalth** [comment]
Active member and park requirements vary from kingdom to kingdom, it needs
to be kingdom configurable.

**Comment by BenOvermyer** [comment]
Alright. That's going to be tricky, then, to do programmatically.

**Comment by amtgard** [comment]
Ben, that's part of the reason that reporting for Active/Contributing member and "average attendance" is so complicated.  It's not a matter of counting records, it's a matter of counting records in accordance with both the rule book and the kingdom bylaws.  Unfortunately, those documents are not technical specifications, they are social contracts.

As per this bug, we can simply have two flags:

One, a per-park flag marker which indicates that the system believes that the park is "Out of Date", similar to the way the front page flags park participation on the front page.

Park activity as currently recorded in the system is intended for administrative use. The kingdom officials of each Kingdom need to be able to move parks on and off their kingdom page based on their needs.  

For instance, let's say we know that Mordengaard is participating every week, but the local PM is not so good.  The "active" flag tells the kingdom and admins that this park exists and is active, whether they are good at reporting attendance or not.  As separate flag can be used for when the system believes the park is truant in reporting; that should be configurable based on "missed weeks per semester", or some other metric or set of configurable metrics.  For now, a hard-coded assumption (like the home page) is probably sufficient.

**Comment by Glenalth** [comment]
Sarah mentioned to me that there is also an issue with parks that just don't use the ORK for record keeping since it's not required, but still want an atlas listing.

I wonder what the pushback would be like at this point if the ORK was to be voted on as the required record keeping solution for Amtgard.

**Comment by Tyedye** [comment]
I don't know of any parks that chose not to use the ORK, but I know it used
to be an use (looking at you, wetlands). Some kingdoms have voted it as
their official kingdom method, but most have not. I think making a
particular tool official is generally a bad idea, particularly since there
is no good punishment for not keeping the records there.

**Comment by BenOvermyer** [comment]
I don't think it should be "required," for exactly the reason you state, Tyedye.

However, use of the ORK should be encouraged, and the best way to encourage that use is to make it a good enough product that it solves more problems than it creates.

In that sense, maybe what the ORK really needs is not a mandate for people to use it, but rather a core, organized team akin to a startup. Not just (pure) developers, but people whose role it is to market it, do customer support, and so forth.

That is not really a discussion for this issue, though. Perhaps a new issue should be created to handle it?
